src/Core/Authorization/AuthorizationResolver.cs

```diff
@@ -525,13 +623,20 @@ private static string GetPolicyWithClaimValues(string policy, Dictionary<string,
     /// <param name="claimsInRequestContext">Dictionary populated with all the user claims.</param>
     /// <returns>The claim value for the given claimTypeMatch.</returns>
```

Comment:
Maybe this comment needs an update. It rather returns the first element/value from the list of values in the claim value.